stetho/src/main/java/com/facebook/stetho/inspector/network/ResponseBodyFileManager.java

```diff
@@ -29,9 +40,15 @@
 public class ResponseBodyFileManager {
   private static final String TAG = "ResponseBodyFileManager";
   private static final String FILENAME_PREFIX = "network-response-body-";
+  private static final int PRETTY_PRINT_TIMEOUT = 10;
```

Comment:
Times are almost always better expressed in Java using milliseconds, and you should also clarify the unit in the variable to avoid confusio (`PRETTY_PRINT_TIMEOUT_MS`)